src/migrations/20210105154127-users-home-region-allow-null.js

```diff
@@ -1,15 +1,9 @@
 module.exports = {
-  up: async (queryInterface, Sequelize) => {
-    await queryInterface.changeColumn('Users', 'homeRegionId', {
-      type: Sequelize.INTEGER,
-      allowNull: true,
-    });
+  up: async () => {
+    // placeholder
```

Comment:
Ah, I ended up modifying this, since there was an issue with undoing the migration. After deleting this file though, there was an error from sequelize looking to run this migration. This info is stored in the SequelizeMeta table.